src/java/org/apache/cassandra/io/util/FileUtils.java

```diff
@@ -1106,4 +1108,11 @@ public static void deleteDirectoryIfEmpty(Path path) throws IOException
             }
         }
     }
+
+    private static ObjectMapper mapper = new ObjectMapper();
```

Comment:
@fibersel would you move this member on top of the class among the other ones please? It should not be here, put seemingly randomly.